WordPress/five-for-the-future
# Manage a Pledge: Enable adding/removing contributors from wp-admin.

This updates the display of contributors into a table view, and adds the ability to add and remove contributors to existing pledges.

The display has been refactored to use JS templates & JSON contributor data– the data is output onto the page when loaded from the server, and rendered when the page finishes loading. Adding & removing contributors now submits to an admin-ajax.php endpoint, which, if successful, return the new list of contributors. This ensures the display is always up to date. 

Fixes #3.

<img width="483" alt="Screen Shot 2019-11-13 at 4 54 19 PM" src="https://user-images.githubusercontent.com/541093/68807714-50186c80-0636-11ea-8bfe-380b5ae82c45.png">

**To test**

- Edit a pledge in wp-admin
- View the contributors, they should be in the correct "confirmed"/"unconfirmed" table
- Resend a confirmation email, it should be sent & you'll get a success alert.
- Remove a contributor, you will have to confirm that you intend to do this.
- Hit cancel, it should not remove the contributor.
- Remove a contributor, hit OK this time
- The list should refresh and that contributor should be gone now
- Add some users to the new contributors text input
- Submit either by hitting enter in the input, or clicking the buttton
- It should succeed, and show those contribs in the unconfirmed list now
- Invalid users should trigger an error message & not add anyone

cc @melchoyce to check design :)
